libmscore/scorefile.cpp

```diff
@@ -428,29 +454,15 @@ bool MasterScore::saveFile()
             // rename old file into backup
             //
             if (dir.exists(basename)) {
-                  if (!dir.rename(basename, backupName)) {
+                  if (!QFile::rename(name, backupDirString + QString(QDir::separator()) + backupName)) {
```

Comment:
QDir::separator() needs to be changed to "/" following the coding rule.

Reply:
Yes, I think so

Reply:
Done.